tests/run/test_named_expressions.py

```diff
@@ -453,7 +453,7 @@ def test_named_expression_scope_19(self):
         def spam(a):
             return a
 
-        res = spam((b := 2))
```

Comment:
Test is copied from CPython test suite and is likely a test that extra brackets are successfully ignored